source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst

```diff
@@ -55,11 +56,15 @@ format is::
 
     mongodb+srv://servername.example.com/{options}
 
+The host name ``servername.example.com`` is the base name to use for SRV and
```

Comment:
if we're already using {options} above, why not something like {hostname}/{address} - I think that would suffice to remove ambiguity here